Nuro/Utilities/CoreData/Repository/RoutineDetailLocalRepository.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  RoutineDetailLocalRepository.swift
+//  Nuro
+//
+//  Created by Karen Natalia on 13/10/22.
+//
+
+import Foundation
+import CoreData
+
+protocol RoutineDetailRepository {
+    func add(dayID: Int, timeID: Int, activity: Activity)
+    func getMaxPosition(routineHeader: RoutineHeader) -> Int16
+    func getRoutineDetails(of header: RoutineHeader) -> [RoutineDetail]
+    func getRoutineDetails(dayID: Int, timeID: Int) -> [RoutineDetail]
+    func updatePosition(routineDetail: RoutineDetail, newPosition: Int)
+    func delete(routineDetail: RoutineDetail)
+}
```

Comment:
_Discussion_: Why not extract all CoreData protocols to another file?
`ex: CoreDataProtocols`